atomic_reactor/constants.py

```diff
@@ -86,7 +86,7 @@
 PLUGIN_BUMP_RELEASE_KEY = 'bump_release'
 PLUGIN_DISTGIT_FETCH_KEY = 'distgit_fetch_artefacts'
 PLUGIN_FETCH_MAVEN_KEY = 'fetch_maven_artifacts'
-PLUGIN_GENERATE_MAVEN_METADATA_KEY = 'generate_maven_metadata'
+PLUGIN_GENERATE_REMOTE_SOURCE_FILES_KEY = 'generate_remote_source_files'
```

Comment:
I'm not entirely sold on this naming, it sounds like it would be related to `resolve_remote_source` but it is not. Is there any way this could be made clearer?

By the way, what exactly *are* remote_source_files? :smile:

Reply:
added type annotations

Reply:
`remote_source_file` is a koji build type we added to attach maven URL source artifacts to brew binary build task that are later fetched by source build and added to source container.